bootstrap.py

```diff
@@ -237,8 +239,18 @@ def get_json(url):
         request.add_header("Authorization", "Basic %s" % base64string)
         result = urllib2.urlopen(request)
         return json.load(result)
+    except urllib2.HTTPError, e:
+        if e.code == 401:
```

Comment:
About HTTPErrors, added a raise. I have not tested the code, but should work.
What do you think?
My thought is that the full try catch need to be rewritten, but this can wait for now.
About return_matching_org_label, that was exactly the meaning of "Some code cleanup is for sure needed" :)